x/subspaces/types/keys.go

```diff
@@ -205,3 +211,35 @@ func SplitUserAddressPermissionKey(key []byte) (subspaceID uint64, sectionID uin
 	user = GetAddressFromBytes(key[lenSubspaceID+lenSectionID:])
 	return subspaceID, sectionID, user
 }
+
+// --------------------------------------------------------------------------------------------------------------------
+
```

Comment:
`ExpiringAllowanceQueuePrefix` for removing expired allowances is impossible to implement in 0.45.11 version of cosmos sdk because `FeeAllowanceI` misses the method called `ExpireAt`. We can implement the feature after updating the cosmos-sdk version over 0.46.

Reference: [0.46.0 FeeAllowanceI](https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/x/feegrant/fees.go#L29) and [0.45.11 FeeAllowanceI](https://github.com/cosmos/cosmos-sdk/blob/v0.45.11/x/feegrant/fees.go#L9)